server/sonar-server/src/main/java/org/sonar/server/webhook/WebhookQGChangeEventListener.java

```diff
@@ -30,6 +32,8 @@
 import org.sonar.db.component.SnapshotDto;
 import org.sonar.server.qualitygate.changeevent.QGChangeEvent;
 import org.sonar.server.qualitygate.changeevent.QGChangeEventListener;
+import org.sonar.server.qualitygate.changeevent.Trigger;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.server.qualitygate.changeevent.Trigger'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FSONAR-10117%2Flive-measures&issues=AWDW84WGJC-F6PRGSYoC&open=AWDW84WGJC-F6PRGSYoC)

[ISSUE_KEY]: AWDW84WGJC-F6PRGSYoC